### PR TITLE
Dummy data generator

### PR DESCRIPTION
**Pull request by connieyeah**
Submitting dummy data generator and schema for review.

### CODE REVIEW

File: database/faker.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+var space = ['Entire Place', 'Private Room', 'Hotel Room', 'Shared Room'];
+for (var i = 0; i < totalNumber; i++) {
+  forListing.push({
+    homeType: space[getRandomInt(4)],
+    placeDescription: faker.lorem.paragraph()
+  });
+}
+
+var forAmenities = [];
```
Comment (wusihong) [inline database/faker.js:22]: minor comment, but stylistically, but per Hack Reactor style guide, suggest naming collections as a plural noun. "amenities" "listings"

File: database/faker.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+for (var i = 0; i < totalNumber; i++) {
+  forAmenities.push({
+    basic: [faker.lorem.word()],
```
Comment (wusihong) [inline database/faker.js]: (1) for the Amenities data, what's the purpose of embedding the value in an array if it's simply a single string? Aren't you just creating more work for yourself? 
(2) just an observation that this section is essentially hardcoded in terms of length for some of these items. I actually don't think it's a big deal, but if your goal is to test your skills rendering react components with different lengths, you won't be able to do that if you hardcode it here. For example, sometimes,safetyFeatures will return 3 items, sometimes 1 item, then the react component will render 3 lines vs. 1 line. You won't get that here. Just an observation

File: database/faker.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    dining: [faker.lorem.word()],
+    guestAccess: [faker.lorem.word()],
+    logistics: [faker.lorem.word()],
+    bedBath: [faker.lorem.word()],
+    safetyFeatures: [faker.lorem.word()],
+    notIncluded: [faker.lorem.word(), faker.lorem.word()]
+  });
+}
+
+console.log(forAmenities);
```
Comment (wusihong) [inline database/faker.js]: don't forget to delete console logs once you are all done :)

File: database/schema.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+var mongoose = require ('mongoose');
+mongoose.connect('mongod://localhost/listing');
+
+var faker = require ('faker');
```
Comment (wusihong) [inline database/schema.js]: doesn't need faker here if you are producing your data in the faker module?

File: database/schema.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+var db = mongoose.connection;
+db.on('error', console.log.error.bind(console, 'connection error'));
```
Comment (wusihong) [inline database/schema.js]: don't understand this line with "console.log.error.bind(..."

File: database/schema.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+var amenitiesSchema = new mongoose.Schema({
+  id: {type: Number, unique: true},
+  basic: String,
+  facilities: String,
+  dining: String,
+  guestAccess: String,
+  logistics: String,
+  bedBath: String,
+  safetyFeatures: String,
+  notIncluded: [String]
```
Comment (wusihong) [inline database/schema.js]: is this correct syntax for array of strings for mongoose? if so, disregard

**Comment by connieyeah** [comment]
Hey Mario, can you please perform a code review. I fixed the unique key issue and am able to make a successful API call. I also did a preliminary set up for my React environment.

**Pull request merged**